### DISCUSSION TITLE
Empty view.html.php file so I can create a site page with VueJs

### DISCUSSION THREAD

**Discussion by kevincam3** [joomengine/Joomla-Component-Builder#739] [Q&A]
Any ideas for how I could create a very basic view. I still need the page to be controlled by Joomla Group permissions but I'm going to build out the whole page using VueJs to create a search page to search a table in the database so I will still want to make sure that the person also has the permissions to perform the search but the actual search will be done with VueJS

**Comment by Llewellynvdm** [comment]
I have build a number of component with this kind of behavior, but none are public at this point, and the explanation of how this is done is not simple, and yet it is easy to do... 

In our next [JUB meeting](https://github.com/vdm-io/Joomla-Component-Builder/discussions/734) front-end view/forms is part of the topic up for discussion. I would invite you to this conversation... maybe you will get some pointers there.

**Reply by kevincam3** [reply]
Sounds good. What I ended up doing was creating a custom dynamic get that just has this one line:

return true.

**Comment by cpaschen** [comment]
Anyone know if this (return true) is the best way to have a custom front-end view without a dynamic get?
